front/app/containers/CustomPageShow/CustomPageProjectsAndEvents.tsx

```diff
@@ -55,23 +70,26 @@ const CustomPageProjectsAndEvents = ({ page }: Props) => {
   return (
     <>
       {page.attributes.projects_enabled && (
-        <ContentContainer>
+        <ProjectCardsContentContainer mode="page">
           <ProjectAndFolderCardsInner
             statusCounts={statusCountsWithoutFilters}
             publicationStatusFilter={publicationStatusFilter}
-            showTitle={false}
+            showTitle={true}
```

Comment:
But the design doesn't have any title. Also, we already have the top info section. Both section and title look duplicated.

Reply:
We can question the design :-) but I've removed it for now. What made me remove it: it'll be the same as on the home page, but maybe admins don't want it to be the same.